src/workloads/query/multiplanner/BlockingSort.yml

```diff
@@ -395,10 +395,10 @@ AutoRun:
   - When:
       mongodb_setup:
         $eq:
-          - standalone-sbe
-          - standalone-80-feature-flags # At time of writing this will enable PM-3591.
-          - standalone-all-feature-flags # At time of writing this will enable PM-3591.
-          - standalone-classic-query-engine
+          - replica
           - replica-all-feature-flags
+          - replica-query-engine-sbe
+          - replica-query-engine-classic
+          - standalone-80-feature-flags # At time of writing this will enable PM-3591.
```

Comment:
I think you should drop the "At time of writing" comment. [SPM-3591](https://jira.mongodb.org/browse/SPM-3591) is fully enabled in master and 8.0, so the comment has grown stale. Same for the changes in all the other workload files.